src/nav/root.yml

```diff
@@ -3,57 +3,57 @@ path: /
 rootNav: true
 pages:
   - title: Welcome to New Relic
-    path: /docs/using-new-relic
+    path: /docs/using-new-relic/welcome-new-relic/get-started/intro-new-relic
   - title: New Relic One
-    path: /docs/new-relic-one/use-new-relic-one
+    path: /docs/new-relic-one/use-new-relic-one/get-started/introduction-new-relic-one
   - title: Guides and best practices
-    path: /docs/new-relic-solutions
+    path: /docs/new-relic-solutions/overview
   - title: section-break
   - title: Alerts and Applied Intelligence
-    path: /docs/alerts-applied-intelligence
+    path: /docs/alerts-applied-intelligence/overview
   - title: Application monitoring (APM)	 		
-    path: /docs/apm
+    path: /docs/apm/new-relic-apm/getting-started/introduction-apm
   - title: Browser monitoring
-    path: /docs/browser
+    path: /docs/browser/browser-monitoring/getting-started/introduction-browser-monitoring
   - title: CodeStream
     path: /docs/codestream/start-here/what-is-codestream
   - title: Distributed tracing
     path: /docs/distributed-tracing
   - title: Errors inbox
     path: /docs/errors-inbox/errors-inbox
   - title: Infrastructure monitoring
-    path: /docs/infrastructure
+    path: /docs/infrastructure/infrastructure-monitoring/get-started/get-started-infrastructure-monitoring/
   - title: Kubernetes and Pixie
-    path: /docs/kubernetes-pixie
+    path: /docs/kubernetes-pixie/docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration
   - title: Log management
-    path: /docs/logs
+    path: /docs/logs/get-started/get-started-log-management
   - title: MLOps
     path: /docs/mlops/get-started/intro-mlops
   - title: Mobile monitoring
-    path: /docs/mobile-monitoring
+    path: /docs/mobile-monitoring/new-relic-mobile/get-started/introduction-mobile-monitoring
   - title: Network monitoring
     path: /docs/network-performance-monitoring/get-started/npm-introduction
   - title: Serverless monitoring
-    path: /docs/serverless-function-monitoring
+    path: /docs/serverless-function-monitoring/overview
   - title: Service Levels management
     path: /docs/service-level-management/intro-slm 
   - title: Synthetic monitoring
-    path: /docs/synthetics
+    path: /docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring/
   - title: section-break
   - title: Account and user management
     path: /docs/accounts/accounts-billing/general-account-settings/introduction-account-settings
   - title: Data and APIs
-    path: /docs/data-apis
+    path: /docs/data-apis/get-started/nrdb-horsepower-under-hood
   - title: Data dictionary
     path: /attribute-dictionary
   - title: Glossary
     path: /docs/glossary/glossary
   - title: 'Release notes'
     path: '/docs/release-notes'
   - title: Security and privacy
-    path: /docs/security
+    path: /docs/security/overview
   - title: Licenses
-    path: /docs/licenses
+    path: /docs/licenses/overview
```

Comment:
@paperclypse, you have several "overview.mdx" files at the parent nodes (the same level as landing pages). I thought we were moving away to the model where the default page for every section is the first child below the top node.